Doc/source/optional.rst

```diff
@@ -1,142 +1,112 @@
 
 .. code:: sh
 
     pip install fonttools[ufo,lxml,woff,unicode]
 
-This command will install fonttools, as well as the optional dependencies that
-are required to unlock the extra features named "ufo", etc.
-
-.. note::
-
-    Optional dependencies are detailed by module in the list below with the ``Extra`` setting that automates ``pip`` dependency installation when this is supported.
+This command installs FontTools along with the optional dependencies required for features like "ufo", "lxml", "woff", and "unicode".
 
+.. note:: sh
 
+    Optional dependencies are listed by module below, along with the `Extra` setting that automates `pip` dependency installation when supported.
 
 :py:mod:`fontTools.misc.etree`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------
 
-The module exports a ElementTree-like API for reading/writing XML files, and allows to use as the backend either the built-in ``xml.etree`` module or `lxml <https://lxml.de>`__. The latter is preferred whenever present, as it is generally faster and more secure.
+This module provides an ElementTree-like API for reading and writing XML files. It supports two backend options:
 
-*Extra:* ``lxml``
+- Built-in `xml.etree` module
+- `lxml` preferred for improved speed and security.
 
+*Extra:* `lxml`
 
 :py:mod:`fontTools.ufoLib`
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Package for reading and writing UFO source files; it requires:
+----------------
 
-* `fs <https://pypi.org/pypi/fs>`__: (aka ``pyfilesystem2``) filesystem abstraction layer.
+Package for reading and writing UFO source files. Dependencies include:
 
-* `enum34 <https://pypi.org/pypi/enum34>`__: backport for the built-in ``enum`` module (only required on Python < 3.4).
-
-*Extra:* ``ufo``
+- `fs <https://pypi.org/pypi/fs>`__: (aka ``pyfilesystem2``) filesystem abstraction layer
+- `enum34 <https://pypi.org/pypi/enum34>`__ backport for the built-in ``enum`` module (module for Python versions < 3.4)
 
+*Extra:* `ufo`
 
 :py:mod:`fontTools.ttLib.woff2`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Module to compress/decompress WOFF 2.0 web fonts; it requires:
+---------------------
 
-* `brotli <https://pypi.python.org/pypi/Brotli>`__: Python bindings of the Brotli compression library.
+Module for compressing and decompressing WOFF 2.0 web fonts. Requires:
 
-*Extra:* ``woff``
+- `brotli <https://pypi.python.org/pypi/Brotli>`__: Python bindings for the Brotli compression library.
 
+*Extra:* `woff`
 
 :py:mod:`fontTools.unicode`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------
 
-To display the Unicode character names when dumping the ``cmap`` table
-with ``ttx`` we use the ``unicodedata`` module in the Standard Library.
+Displays Unicode character names when working with the `cmap` table dumps,with `ttx` we use the `unicodedata` module in the Standard Library. 
```

Comment:
There's a space missing between "," and "with". I would also prefer to move towards "FontTools uses the `unicodedata` [...] over "we use [...]" but that might be a bigger discussion. Certainly it's not _wrong_ to say "FontTools" instead of "we" though. Also, I would drop the "the" before `cmap`; pretty minor though.